docs/design/3_1_5_communications.tex

```diff
@@ -1 +1,35 @@
-TODO: Why WiFi? Do we need an external router? Throughput/bandwidth considerations? Why not combine it with drone control?
\ No newline at end of file
+The Air-Ground communication between the Raspberry Pi and the Base Station transmits video, ML results and control commands (to the computing platform). It is a bi-directional, high bandwidth communication channel that requires stable connection range for up to 30 meters.
+
+There are two options for establishing this communication - single channel and dual channel. The first option is using the existing communication channel, the one used for drone controlling, to transmit data. The advantages are it doesn't require additional hardware and save power. The disadvantage is that the drone control is using an off-the-shelf module, in order to extend its functionality to support transmit more data through it, we need to make modifications to a highly integrated system. The second option is having a separate Air-Ground communication channel. This channel can be established under commonly used wireless communication methods(i.e. Wi-Fi or Bluetooth). It requires a dedicated pair of transceivers. The advantage of this option is that most development boards are equipped with wireless transceivers and the communication is easily configured.
+
+We chose the second option as it is more suitable for product development.
+
+\textbf{Bandwidth Requirement}
+
+The majority of the bandwidth will be reserved for video transmission as it's the most intensive transmission task among all the tasks. The total number of bits per second of video that need to be transmitted can be calculated using the following formula:
+
+\[ \#\:of\:bits/second = \dfrac{\#\:of\:bits}{1\:pixel} * \dfrac{\#\:of\:pixels}{1\:frame} * \dfrac{\#\:of\:frames}{1\:second} \]
+
+The video that will be transmitted has a resolution of 640x480 pixels and is 30 frames per second. The colour depth is 24-bit for a RGB colour set. That is, each pixel is 8 bits. Then:
+
+\[ \#\:of\:bits/second = \dfrac{8}{1} * \dfrac{640*480}{1} * \dfrac{30}{1} = 73.728\:Mbit/s \]
+
+The other data that need to be transmitted includes the metadata of the video, the results from ML model, the control commands, and required field for the protocol. The overall bandwidth required is about 75 Mbit/s.
```

Comment:
You should mention that this is assuming we don't do any video compression (i.e., this is our worst case). Of course, if the WiFi can handle this, that's great, seeing how it's still unclear as to how much processing power we can afford for video compression/decompression.